packages/indexer-cli/src/commands/indexer/actions/approve.ts

```diff
@@ -48,7 +51,18 @@ module.exports = {
         throw Error(`Missing required argument: 'actionID'`)
       }
 
-      numericActionIDs = actionIDs.map(action => +action)
+      // If actionIDs is 'queued', then populate actionIDs with actions that are queued
+      if (actionIDs.join() == 'queued') {
+        const queuedActions = await fetchActions(client, {
+          status: ActionStatus.QUEUED,
+        })
+        numericActionIDs = queuedActions.map(action => action.id)
+      } else {
+        numericActionIDs = actionIDs.map(action => +action)
+      }
+      if (numericActionIDs.length === 0) {
+        throw Error(`Failed to process action ids, check queue for available actions`)
```

Comment:
Not sure users are going to immediately know what went wrong and how to debug from this log message.

What if we move this check into the first if block of the if..else statement above and make it specific to the `queued` actions? (The case of an empty `actionIDs` array is already checked for above.) 

We could then simply log:
```
`No 'queued' actions found.`
```